fendermint/vm/interpreter/src/fvm/state/query.rs

```diff
@@ -182,6 +182,14 @@ where
                     });
                 }
             }
+
+            // If the gas_limit is zero, set it to the block gas limit so that call will not hit
+            // gas limit not set error. It is possible, in the future, to estimate the gas limit
+            // based on the account balance and base fee + premium for higher accuracy.
```

Comment:
Currently it's possible to set the gas prices to zero. Ethers does the estimation if you ask it, but I reckon we shouldn't complicate _this_ handler with estimations. That kind of shenanigans should really go into the Ethereum API.

Reply:
hahaha there was a race in the review. I left a similar comment below, but you can disregard it.